rules/library.bzl

```diff
@@ -468,31 +490,8 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
     swift_libname = "%s_swift" % name
     cpp_libname = "%s_cpp" % name
 
-    # TODO: remove framework if set
-    if namespace_is_module_name and not module_map and \
-       (objc_hdrs or objc_private_hdrs or swift_sources or objc_sources or cpp_sources):
-        umbrella_header = library_tools["umbrella_header_generator"](
-            name = name,
-            library_tools = library_tools,
-            public_headers = objc_hdrs,
-            private_headers = objc_private_hdrs,
-            module_name = module_name,
-            **kwargs
-        )
-        if umbrella_header:
-            objc_hdrs += [umbrella_header]
-        module_map = library_tools["modulemap_generator"](
-            name = name,
-            library_tools = library_tools,
-            umbrella_header = paths.basename(umbrella_header),
-            public_headers = objc_hdrs,
-            private_headers = objc_private_hdrs,
-            module_name = module_name,
-            framework = False if swift_sources else True,
-            **kwargs
-        )
-
     if swift_sources:
+        swift_copts.extend(("-Xcc", "-I."))
```

Comment:
what does `-I.` do again? Since it is used in multiple places down there, wonder if creating a macro representing this in this file will make it more readable

Reply:
`-I.` adds the current directory to the header search paths, and it's needed _after_ the `-I` flags pointing to the hmaps, otherwise relative paths inside hmaps don't resolve

Reply:
so to be clear `.` is for the current directory? Wonder if this is useful info to be included as a comment

Reply:
Not sure if you've already done this, but if not, I'd recommend looking at `swiftc --help` and `man clang` for what these args are for. There is also documentation up at https://clang.llvm.org/docs/

Reply:
@segiddins What is the "current directory" that -I. refers to? Is it the directory of the BUILD file that calls this macro or another?